Sources/RxAlamofire.swift

```diff
@@ -775,11 +775,11 @@ extension Reactive where Base: DataRequest {
                 case .success(let result):
                     if let httpResponse = packedResponse.response {
                         observer.on(.next(httpResponse, result))
+                        observer.on(.completed)
                     }
                     else {
                         observer.on(.error(RxAlamofireUnknownError))
                     }
-                    observer.on(.completed)
```

Comment:
Yeah, I was thinking about it as well. The problem here is that I'm not sure if there is a way to distinguish if this is streaming connection. 
For example I ones used it for [Firebase](https://firebase.google.com/docs/reference/rest/database/#section-streaming) (because they do not support tvOS) and the only difference is in `Accept` header field